### PR TITLE
docs(oss-2.5.x) regenerate autodocs for 2.5

### PR DESCRIPTION
**Pull request by kikito**
### Summary

The Admin API now includes both the db and db-less modules in a single
one. These changes include:

* Putting declarative and db-less sections at the top
* Marking the methods that are not compatible with DB-less as such

The "movement" at the top of the file is due to a new version of
openresty and cannot be avoided easily. It is recommended to review
the diff with "ignoring blank space differences" as some end-of-line
spaces were removed.

### Reason

In preparation for 2.5

### CODE REVIEW

File: app/gateway-oss/2.5.x/admin-api.md [hunk 1]
```diff
@@ -1172,6 +1255,11 @@ Services can be both [tagged and filtered by tags](#tags).
 
 ### Add Service
 
+
```
Comment (HCloward) [inline app/gateway-oss/2.5.x/admin-api.md:1258]: We have changed how we add admonitions recently. See the [Admonitions](https://docs.konghq.com/contributing/markdown-rules/#admonitions) section in the Markdown rules page of our Contribution guidelines.

File: app/gateway-oss/2.5.x/admin-api.md [hunk 2]
```diff
@@ -1172,6 +1255,11 @@ Services can be both [tagged and filtered by tags](#tags).
...
+<div class='alert alert-info.blue' role='alert'>
```
Comment (HCloward) [inline app/gateway-oss/2.5.x/admin-api.md:1259]:
```suggestion
{:.note} 
**Note:** Not available in DB-less mode.
--



```

File: app/gateway-oss/2.5.x/admin-api.md [hunk 3]
```diff
@@ -1172,6 +1255,11 @@ Services can be both [tagged and filtered by tags](#tags).
...
+Not available in DB-less mode
```
Comment (HCloward) [inline app/gateway-oss/2.5.x/admin-api.md:1260]:
```suggestion

```

File: app/gateway-oss/2.5.x/admin-api.md [hunk 4]
```diff
@@ -1172,6 +1255,11 @@ Services can be both [tagged and filtered by tags](#tags).
...
+</div>
```
Comment (HCloward) [inline app/gateway-oss/2.5.x/admin-api.md:1261]:
```suggestion

```

File: autodoc-nav/docs_nav_2.5.x.yml.head.in [hunk 5]
```diff
@@ -0,0 +1,250 @@
+# Generated via autodoc/admin-api/generate.lua
+
```
Comment (HCloward) [inline autodoc-nav/docs_nav_2.5.x.yml.head.in:2]: Seems like this is missing a section here . . . the title, url, and items?

Comment (HCloward) [reply autodoc-nav/docs_nav_2.5.x.yml.head.in:2]:
```suggestion
- title: Admin API
   url: /admin-api/
   items:
```

**Review by HCloward — CHANGES_REQUESTED** [review]
I pulled down your changes and build locally and noticed the new section links in the left nav to not work - I think it may have to do with the missing section in the docs_nav_2.5.x.yml.head.in file. I made a comment. Also, we updated how we format markdown admonitions. I linked to the explanation and updated one instance in your doc. Thanks for making these updates =)

**Comment by kikito** [comment]
Hi @HCloward , I have done the requested changes and regenerated, please give this another review when possible.

**Comment by HCloward** [comment]
> Hi @HCloward , I have done the requested changes and regenerated, please give this another review when possible.

Hi @kikito everything looks good. After talking with the other technical writers we thought a badge might be a nice addition for db-less compatible. I added it to the sections that are compatible with db-less. Let me know what you think. Also, for some reason the navigation links in the left nav still do not work, even after you updated the docs_nav_2.5.x.yml.head.in file.

**Comment by kikito** [comment]
I believe I have fixed the navigation (hopefully, I can't try locally). I have also added the dbless badges to all the methods that didn't have the "This does not work on dbless" warning.

@HCloward please give this another review when you can

**Review by HCloward — APPROVED** [review]
Made a final update to the admin api doc that moves the badges inline with the headings. Lena fixed it in an earlier note - but it got lost when we force pushed. Thanks Enrique!

**Comment by kikito** [comment]
@HCloward I changed the generator to produce the badges and re-run it.

There is one difference: the newlines are produced in a way that is difficult for me to change on the generator, so instead of

```
## header
{:.badge .dbless}
```

It is often
```
## header


{:.badge .dbless}
```

I hope this is acceptable.

**Comment by HCloward** [comment]
> @HCloward I changed the generator to produce the badges and re-run it.
> 
> There is one difference: the newlines are produced in a way that is difficult for me to change on the generator, so instead of
> 
> ```
> ## header
> {:.badge .dbless}
> ```
> 
> It is often
> 
> ```
> ## header
> 
> 
> {:.badge .dbless}
> ```
> 
> I hope this is acceptable.

So I ran it on my local and it ends up adding the badge to the first line of text instead of the heading. So perhaps I think the old way was better. At least it wasn't next to the first line of text.

**Comment by kikito** [comment]
I found a way to generate badges cleanly without the extra blank lines. I think it looks better this way, so I have reverted the html. I hope this is the definitive one.

**Comment by HCloward** [comment]
> I found a way to generate badges cleanly without the extra blank lines. I think it looks better this way, so I have reverted the html. I hope this is the definitive one.

Built locally and it looks perfect - thank you so much for your patience @kikito !

**Pull request merged**